zio-config/src/main/scala/zio/config/actions/Write.scala

```diff
@@ -8,6 +8,9 @@ final case class Write[A](run: ZIO[A, WriteError, Map[String, String]])
 object Write {
   final def write[A](config: Config[A]): Write[A] =
     config match {
+      case Config.Pure(_) =>
+        Write(ZIO.access(_ => Map.empty))
```

Comment:
Why is the pure value not used here?

Reply:
Config of pure describes no effects. It only lifts. Config.Pure(a) doesn’t mean it needs to read anything from ConfigSource, nor it says there is nothing to write given an A